ui/src/app/edge/history/history.module.ts

```diff
@@ -66,10 +61,7 @@ import { TimeOfUseTariffDischargeWidgetComponent } from './timeofusetariffdischa
 @NgModule({
   imports: [
     SharedModule,
-    Common_Autarchy,
-    Common_Production,
-    Common_Selfconsumption,
-    CommonEnergyMonitor
+    Common_Module

```

Comment:
Usually you would not use UpperCamelCase and Underscore for naming. Why not `CommonModule` or only `Common`?

Reply:
Common then